packages/metric/README.md

```diff
@@ -73,6 +72,66 @@ export class TestClassController {
 
 export class AppModule {}
 ```
+### MetricModule
+The MetricModule provides functionality for working with metrics in your application. 
+It adds the MetricService to the application using the IMetricService token and GraphiteService using the IGraphiteService token.
+ConfigModule is required for MetricModule to work
+
+### Imports MetricModule and ConfigModule
+```typescript
+import { Module } from '@nestjs/common'
+import { MetricModule} from '@qiwi/nestjs-enterprise-metric'
+import { ConfigModule } from '@qiwi/nestjs-enterprise-config'
+
+@Module({
+  imports: [
+    MetricModule, 
+    ConfigMoudle,
+  ],
+})
+export class AppModule {}
+```
+In this case you should specify parameters in config file
+```json
+{
+    "data": {
+      "name": "test-name-app",
+
+      "graphite": {
+        "url": "http://graphite-url.com"
+      },
+
+      "metric":{
+        "prefix": "metric",
+        "interval": 30
+      },
+  }
+}
+```
+
+### Imports MetricModule and ConfigModule with parameters 
+
+```typescript
+import { Module } from '@nestjs/common'
+import { MetricModule} from '@qiwi/nestjs-enterprise-metric'
+import { ConfigModule } from '@qiwi/nestjs-enterprise-config'
+
+@Module({
+  imports: [
+    MetricModule.register({
+      graphiteApiEndpoint: 'https://localhost:8080',
+      metricsConfig: { prefix: 'some-prefix'; interval: 5000 },
+    }),
+    ConfigModule.register({ path: 'some/path' })
+  ],
+})
+export class AppModule {}
+```
+You can pass parameters to a `MetricModule.register()` method
+- graphiteApiEndpoint - The URL for connecting to the Graphite API
```

Comment:
graphiteUrlOrService - URL for connecting to the Graphite API or your own implementation of IGraphiteService